scripts/keep.py

```diff
@@ -1,10 +1,16 @@
-from brownie import accounts, network, interface, Vault
+from brownie import accounts, network, interface, Vault, Token
+from brownie.network.gas.strategies import GasNowScalingStrategy
 from decimal import Decimal
 from eth_utils import is_checksum_address
 import requests
 from time import sleep
 
 
+GAS_BUFFER = 1.2
+
+gas_strategy = GasNowScalingStrategy()
```

Comment:
Ideally we need a custom logic to rather cancel the tx for 21k gas on gas prices spike.

Reply:
have to add to brownie